templates/deploy-to-existing-kubernetes-cluster.yml

```diff
@@ -13,7 +13,7 @@ variables:
   # Container registry service connection established during pipeline creation - Docker Hub/ACR/...
   dockerRegistryServiceConnection: '{{ containerRegistryConnection.Id }}'
   imageRepository: '{{ imageRepository }}'
-  containerRegistryName: '{{ containerRegistryName }}'
+  containerRegistry: '{{ containerRegistryConnection.Authorization.Parameters.loginServer }}'

```

Comment:
can you please remove the parameter from props file in that case?

Reply:
This is not added to properties.json file